felix/fv/bpf_test.go

```diff
@@ -86,11 +86,11 @@ var (
 	_ = describeBPFTests(withTunnel("ipip"), withProto("udp"))
 	_ = describeBPFTests(withProto("tcp"), withDSR())
 	_ = describeBPFTests(withProto("udp"), withDSR())
-	_ = describeBPFTests(withTunnel("ipip"), withProto("tcp"), withDSR())
-	_ = describeBPFTests(withTunnel("ipip"), withProto("udp"), withDSR())
+	_ = describeBPFTests(withTunnel("ipip"), withProto("tcp"), withDSR(), withConntrackCleanupMode("BPFProgram"))
```

Comment:
Any benefit of forcing BPFProgram? In our test env, it should always use BPFProgram. Unless you want it to fail in some envs where it does not load?

Reply:
I wanted to force it so that the test would fail if the program failed to load.